### PR TITLE
Enable tests / fix jshint/jscs

### PR DESCRIPTION
**Pull request by bcipolli**
Running npm tests locally wasn't working well, due to the large # of jshint/jscs failures and a brittle test for an API endpoint.

I updated the code, some of the jshint/jscs settings, and the test, then added `npm test` to the Travis config file, so that hopefully future PRs will maintain working tests and get auto-style-checked.

### CODE REVIEW

File: app/components/common/cityModule/cityElectionsController.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 'use strict';
 
 function cityElectionsController($scope, ballot) {
-  $scope.ballot = ballot;
-
-  $scope.isOffice = isOffice;
-  $scope.isReferendum = isReferendum;
-
```
Comment (adborden) [inline app/components/common/cityModule/cityElectionsController.js]: This is here as per @jnmarcus 's styleguide. https://github.com/johnpapa/angular-styleguide#bindable-members-up-top

But instead you should disable the `latedef` jshint option.

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "Disclosure-Frontend-Alpha",
   "version": "1.0.0",
   "description": "Base AngularJS application with gulp and browserify",
+  "license" : "MIT",
```
Comment (adborden) [inline package.json]: Let's leave this out for now, I'd like to pitch AGPL.

File: app/components/services/disclosure/index.spec.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ describe('disclosureApi', function() {
   }));
 
   it('exists', function() {
-    expect(disclosureApi).to.be.ok;
+    expect(disclosureApi).to.be.ok;  // jshint ignore:line
```
Comment (adborden) [inline app/components/services/disclosure/index.spec.js]: `/*jshint expr: true*/` should just be at the top of every test file.

File: .travis.yml [hunk 4]
```diff
@@ -17,3 +17,6 @@ deploy:
   skip_cleanup: true
   on:
     branch: master
+
+script:
+  - npm test
```
Comment (adborden) [inline .travis.yml]: This isn't necessary, travis detects this as a node project and runs `npm test` by default. The style checks haven't been integrated into `npm test` yet.

**Comment by bcipolli** [comment]
@adborden Thanks for the tips! Hope I can be more helpful for front-end things in the future with less hand-holding... but I appreciate it!

Will update this tomorrow-ish.

**Comment by adborden** [comment]
@bcipolli I think I addressed most of the linting issues. Should be clean on master.

**Pull request closed without merging**